### PR TITLE
[T4A3][T11-B1] Tan Zhi Jie

### PR DESCRIPTION
**Pull request by zhijietan94**

### CODE REVIEW

**Comment by zhijietan94** [comment]
Edited the pull request name from T11-B01 to T11-B1

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
```
Comment (xdshang) [inline src/seedu/addressbook/data/person/Contact.java:3]: Need a head comment for this class.

File: src/seedu/addressbook/data/person/Address.java [hunk 2]
```diff
@@ -6,53 +6,30 @@
...
-        return test.matches(ADDRESS_VALIDATION_REGEX);
-    }
-
-    @Override
-    public String toString() {
-        return value;
-    }
-
     @Override
     public boolean equals(Object other) {
```
Comment (xdshang) [inline src/seedu/addressbook/data/person/Address.java:29]: Usually `equals(...)` and `hashCode()` are working together, so it is recommended that you abstract both of them or not at the same time.

File: src/seedu/addressbook/data/person/Address.java [hunk 3]
```diff
@@ -6,53 +6,30 @@
...
      * @throws IllegalValueException if given address string is invalid.
      */
     public Address(String address, boolean isPrivate) throws IllegalValueException {
         this.isPrivate = isPrivate;
-        if (!isValidAddress(address)) {
+        if (!isValidContact(address, ADDRESS_VALIDATION_REGEX)) {
             throw new IllegalValueException(MESSAGE_ADDRESS_CONSTRAINTS);
         }
         this.value = address;
     }
```
Comment (xdshang) [inline src/seedu/addressbook/data/person/Address.java:26]: The constructor could be abstracted as well.

**Comment by xdshang** [comment]
@zhijietan94 Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**